drivers/Ticker.h

```diff
@@ -76,21 +76,12 @@ class Ticker : public TimerEvent, private NonCopyable<Ticker> {
 
     /** Attach a function to be called by the Ticker, specifying the interval in seconds
      *
-     *  The method forwards its arguments to attach_us() rather than copying them which
-     *  may not be trivial depending on the callback copied.
-     *  The function is forcibly inlined to not use floating-point operations. This is
-     *  possible given attach_us() expects an integer value for the callback interval.
      *  @param func pointer to the function to be called
      *  @param t the time between calls in seconds
      */
-#if defined(__ICCARM__)
-    MBED_FORCEINLINE template <typename F>
-#else
-    template <typename F> MBED_FORCEINLINE
-#endif
-    void attach(F &&func, float t)
```

Comment:
Why is the forwarding being dropped? It reduced code size. Does it have any bearing on the `float` elimination?

Reply:
No it does not. I was checking if float elimination needed that. I will put it back for code size reduction.